### ISSUE TITLE
Need rewards plan

### ISSUE THREAD

**Issue by bots-business** [bots-business/BBExpertsList#24]
We have such reward draft:

![image](https://user-images.githubusercontent.com/38240269/118970461-99771400-b9b1-11eb-956e-fbdad5a5f6b6.png)

Need plan about BB Points and BB Token rewards.

**Prize fund:**
- 100 000 BBP
- 100 000 BB

**Others possible funds:**
- something other? Chat Titles?
- other ideas?

Need details who will be rewarded, amount and etc.

**Comment by killerbhai** [comment]
Rewards depend on type of help

**Comment by mohdradin** [comment]
What Plan

**Comment by bots-business** [comment]
We have

Prize fund:

100 000 BBP
100 000 BB


Plan:
50 000 BBP - top 1 Expert
30 000 BBP - top 2 Expert
etc...

or other ideas?

**Comment by TEAMKONG1** [comment]
Hi how can I be in the list of experts

**Comment by bots-business** [comment]
### We need minimum 20-100 EL or ML per  Expert.
Because lower values is not trusted!

More EL+ML - Prize Fund will be reached to max limits to: 
**100K BBP + 100K BB - max limit** (but we need end date for Expert Quest)

### current Prize Fund limits
We have 30 Level points total.
30 * 10 = 300.
10 - it is reward for 1 Level (can be increased)

**Current Prize Funds:**
- 300 BBP
- 300 BB




### How to increase Prize Funds?
- be active!
- invite new Experts!
- earn EL + ML 
- more EL+ML - more Prize Funds 
- help to another Experts!

**Comment by sohamdeogaonkar** [comment]
**We Could divide these funds in 2 groups**

1) To the top Users of the Season
2) Other Fund to the Remaining Users 

We could have the seasons, Lasting 3-4 months every

If we plan this for a years
We could have fund of 25K for every season

Instead of the Expert level 
We could have the User Experience system

We could give experience based on there help to bb.

**So from 25K , we could give**
- Top Expert - 10K
- 2nd Expert - 7K
- 3rd Expert - 5K

so we left 3K fund more
so we can divide them in other remaining users according to there experience

**Comment by bots-business** [comment]
> **We Could divide these funds in 2 groups**
> 
> 1. To the top Users of the Season
> 2. Other Fund to the Remaining Users
> 
> We could have the seasons, Lasting 3-4 months every
> 
> If we plan this for a years
> We could have fund of 25K for every season
> 
> Instead of the Expert level
> We could have the User Experience system
> 
> We could give experience based on there help to bb.
> 
> **So from 25K , we could give**
> 
> * Top Expert - 10K
> * 2nd Expert - 7K
> * 3rd Expert - 5K
> 
> so we left 3K fund more
> so we can divide them in other remaining users according to there experience

+1 ML

**Comment by ShadabAlam786** [comment]
> > **We Could divide these funds in 2 groups**
> > 
> > 1. To the top Users of the Season
> > 2. Other Fund to the Remaining Users
> > 
> > We could have the seasons, Lasting 3-4 months every
> > If we plan this for a years
> > We could have fund of 25K for every season
> > Instead of the Expert level
> > We could have the User Experience system
> > We could give experience based on there help to bb.
> > **So from 25K , we could give**
> > 
> > * Top Expert - 10K
> > * 2nd Expert - 7K
> > * 3rd Expert - 5K
> > 
> > so we left 3K fund more
> > so we can divide them in other remaining users according to there experience
> 
> +1 ML

Yes It's Correct. By This Way All Users Who Is Participated In Contest They All Will Get Happy. And From New Contest More Users Will Try To Participate In Contest.
